MIGRATION.md

```diff
@@ -0,0 +1,86 @@
+0.19 to 0.21
+============
+
+## Configuration
+
+The biggest changes from 0.19 to 0.21 are related to the application.conf file, which has been restructured significantly.
+
+The configuration for backends now is all contained within a `backend` stanza, which specifies 1 stanza per name per backend and a default backend, as follows:
+
+```
+backend {
+    default=Local
+    providers {
+        Local {
+            actor-factory: "class path to BackendLifecycleActorFactory implementation"
+            config {
+                ... backend specific config ...
+            }
+        }
+        JES {
+            actor-factory: "class path to BackendLifecycleActorFactory implementation"
+            config {
+                ... backend specific config ...
+            }
+        }
+        SGE {
+            actor-factory: "class path to BackendLifecycleActorFactory implementation"
+            config {
+                ... backend specific config ...
+            }
+        }
+    }
+}
+```
+
+### Migrating a 0.19 config file to a 0.21 config file
+
+Copy `core/src/main/resources/application.conf` somewhere, and open up your old config file and copy the following values from the 0.19 config into the 0.21 config:
+
+Below is a table based diff, but there is also a [visual diff](http://i.imgur.com/i8j0Zvb.jpg) of the changes.
+
+|0.19|0.21|Comments|
+|----|----|--------|
+||`akka.actor.deployment`|Configures the number of threads to handle workflow log copying|
+||`akka.dispatchers.slow-actor-dispatcher`|Configures the "slow-actor" dispatcher|
```

Comment:
This has been renamed to `io-dispatcher`.  There are other new dispatchers as well.